performancetest/src/main/java/org/corfudb/performancetest/StreamPerformanceTest.java

```diff
@@ -0,0 +1,103 @@
+package org.corfudb.performancetest;
+
+import com.codahale.metrics.MetricRegistry;
+import com.codahale.metrics.Timer;
+import org.corfudb.runtime.CorfuRuntime;
+import org.corfudb.runtime.view.stream.IStreamView;
+import org.junit.Test;
+import java.io.IOException;
+
+/**
+ * Created by Nan Zhang and Lin Dong.
+ */
+
+public class StreamPerformanceTest extends PerformanceTest {
+    static MetricRegistry metricRegistry;
+    Timer producerTimer;
+    Timer consumerTimer;
+    private static final String METRIC_PREFIX = "corfu-perf.";
+    private static final String STREAM_NAME = "stream-perf-it";
+    private static final String OBJECT_NUM = "appendObjectNum";
+    private static final String OBJECT_SIZE = "appendObjectSize";
+    private static final String PRODUCER_NUM = "producerNum";
+
+    public StreamPerformanceTest() {
+        metricRegistry = CorfuRuntime.getDefaultMetrics();
+        producerTimer = metricRegistry.timer(METRIC_PREFIX + "stream-producer");
+        consumerTimer = metricRegistry.timer(METRIC_PREFIX + "stream-consumer");
+    }
+
+    /**
+     * tests append() and next() performance on a stream,
+     * with single producer and single consumer in two threads.
+     *
+     * This test does not check correctness.
+     */
+    @Test
+    public void streamSingleProducer() throws IOException, InterruptedException {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [JUnit tests should include assert() or fail()](https://app.codacy.com/app/annymartinez.m/CorfuDB/pullRequest?prid=4651454)